[AY1920S1-CS2113T-W12-3] DukeFacilityBooking

AmosChan97 opened the pull request:
@wyinnnn
@zkchang97
@AmirAzhar
@Alex-Teo

zkchang97 committed:
Merge pull request #66 from zkchang97/master

Adds ListRoomCommand.java to list the rooms available for booking

Commit:
Merge pull request #6 from AY1920S1-CS2113T-W12-3/master

Update Personal Repo

Alex-Teo committed:
Add ApproveCommand and RejectCommand

Alex-Teo committed:
Add ApproveCommand and RejectCommand

Alex-Teo committed:
Merge branch 'master' into master

Alex-Teo committed:
Merge pull request #8 from AY1920S1-CS2113T-W12-3/master

Merge 7

Alex-Teo committed:
Add (temp) ListRoomCommand

zkchang97 committed:
Merge remote-tracking branch 'upstream/master'

# Conflicts:
#	src/main/java/booking/BookingList.java
#	src/main/java/command/Command.java
#	src/main/java/control/Parser.java
#	src/main/java/room/RoomList.java

zkchang97 committed:
Merge remote-tracking branch 'upstream/master'

# Conflicts:
#	src/main/java/booking/BookingList.java
#	src/main/java/command/Command.java
#	src/main/java/control/Parser.java
#	src/main/java/room/RoomList.java

Alex-Teo committed:
Merge pull request #67 from Alex-Teo/master

[CS2113T-W12-3]-Alex-Teo-Approve-Reject-Commands

Alex-Teo committed:
Merge branch 'master' into master

Alex-Teo committed:
Merge pull request #68 from zkchang97/master

[CS2113T-W12-3]-Zhong Kai-Edits to AddRoomCommand

Alex-Teo committed:
Merge pull request #9 from AY1920S1-CS2113T-W12-3/master

merge 9

Commit:
Merge pull request #7 from AY1920S1-CS2113T-W12-3/master

Update Personal Repo

Commit:
Create LogoutCommand

Commit:
Merge pull request #69 from AmirAzhar/master

Create Logout Command

okkhoy commented on src/main/java/Facility.java:
```diff
@@ -0,0 +1,52 @@
+import java.util.ArrayList;
+
+public class Facility {
+
```
need header comments for the class and the protected members if they have explicit meaning in your codebase

okkhoy commented on src/main/java/Facility.java:
```diff
@@ -0,0 +1,52 @@
+
+    protected String Name;
+
+    protected ArrayList <Items> inventory; //convert to 2d array
```
use github comments like this one instead of the inline comments

okkhoy commented on src/main/java/Facility.java:
```diff
@@ -0,0 +1,52 @@
+    }
+
+    /**
+     * To edit the inventory of the room
```
violating the standards for header comments

okkhoy commented on src/main/java/Main.java:
```diff
@@ -0,0 +1,34 @@
+            fxmlLoader.<MainWindow>getController().setDuke(duke);
+            stage.show();
+        } catch (IOException e) {
+            e.printStackTrace();
```
printing stacktrace is not good exception handling.

okkhoy commented on src/main/java/booking/BookingList.java:
```diff
@@ -0,0 +1,57 @@
+     */
+    public BookingList(ArrayList<String> loader) throws DukeException {
+        for (String line : loader) {
+            String[] splitStr = line.split(" \\| ", 6);
```
violates SLAP

okkhoy commented on src/main/java/booking/BookingList.java:
```diff
@@ -0,0 +1,57 @@
+     * @param timeEnd the ending time
+     * @return if there is already a request with that venue and time slot
+     */
+    public static boolean checkBooking(BookingList bookinglist, String roomcode, String timeStart, String timeEnd) {
```
any particular reason to make this static?

okkhoy commented on src/main/java/command/AddBookingCommand.java:
```diff
@@ -0,0 +1,76 @@
+        }
+
+        String temp = input.substring(4); // name description /at roomcode /from dd/mm/yyyy hhmm /to dd/mm/yyyy hhmm
+        splitC = temp.split(" /at ", 2); // splitC[] = {name, description, roomcode, dd/mm/yyyy hhmm /to dd/mm/yyyy hhmm)
```
this is gross violation of SLAP.  you are doing parsing and low-level string manipulations in the same class

okkhoy commented on src/main/java/command/AddRoomCommand.java:
```diff
@@ -0,0 +1,64 @@
+        String tempAR = input.substring(8);
+        splitC = tempAR.split(" /date ", 2); // splitC[] = {ROOMCODE, DATE /timeslot TIMESLOT}
+        this.roomcode = splitC[0]; // ROOMCODE
+        this.datesplit = splitC[1].split(" /timeslot ", 2); // datesplit[] == {DATE, TIMESLOT}
```
same comment as above

okkhoy commented on src/main/java/command/ApproveCommand.java:
```diff
@@ -0,0 +1,60 @@
+        splitC = input.split(" ", 5);
+        name = splitC[1];
+        roomcode = splitC[2];
+        if(!RoomList.checkRoom(roomcode,"data\\roomlist.txt")) {
```
don't hardcode file paths. use configuration files when necessary

okkhoy commented on src/main/java/command/ByeCommand.java:
```diff
@@ -0,0 +1,30 @@
+public class ByeCommand extends Command {
+
+    /**
+     * If "bye" is entered.
```
violates standards for header comments

okkhoy commented on src/main/java/command/Command.java:
```diff
@@ -0,0 +1,70 @@
+    }
+
+
+    public void execute (RoomList roomlist, Ui ui, Storage roomStorage) throws DukeException, IOException, ParseException {
```
what is this method? why is it different from the above?

okkhoy commented on src/main/java/command/CreateAccountCommand.java:
```diff
@@ -0,0 +1,54 @@
+     */
+    public CreateAccountCommand(String input, String[] splitStr) throws DukeException, IOException {
+        File membersFile = new File("data\\members.txt");
+        membersFile.createNewFile(); //if file already exists, won't create
```
refer to the earlier comments on SLAP

okkhoy commented on src/main/java/command/EditBookingCommand.java:
```diff
@@ -0,0 +1,60 @@
+        if(!RoomList.checkRoom(roomcode,"data\\roomlist.txt")) {
+            throw new DukeException("\u2639 OOPS!!! This room doesn't exist!");
+        }
+        datetimeStartString = splitC[3] + " " + splitC[4];
```
why 3 and 4? why not 5 and 6? (read as magic numbers)

okkhoy commented on src/main/java/command/ListRoomCommand.java:
```diff
@@ -0,0 +1,26 @@
+import user.User;
+
+
+public class ListRoomCommand extends Command {
```
no header comments

okkhoy commented on src/main/java/control/Parser.java:
```diff
@@ -0,0 +1,49 @@
+package control;
+
+import command.*;
```
don't import everything using *, import necessary classes explicitly.

okkhoy commented on src/main/java/control/Parser.java:
```diff
@@ -0,0 +1,49 @@
+package control;
```
Many classes seem to be in a single package control? (at least thus far) think a bit about SOC and how you want to organize the code.

okkhoy commented on src/main/java/room/RoomList.java:
```diff
@@ -0,0 +1,37 @@
+        String line = reader.readLine();
+
+        while (line != null){
+            if(line.equals(roomcode))
```
need to have braces. even for single line conditionals

okkhoy commented on src/main/java/task/FixedDuration.java:
```diff
@@ -0,0 +1,34 @@
+    protected String needs;
+
+    /**
+     * task.FixedDuration constructor
```
incorrect format for header comments

okkhoy commented on src/main/java/user/Login.java:
```diff
@@ -0,0 +1,98 @@
+
+import ui.Ui;
+
+import java.io.*;
```
dont import all using *

okkhoy reviewed (commented):
Overall, I think you have a bit of work to reorganize code. I found quite a few violations of SLAP.
It is good to see you have started something on the test code front. Please ensure you have written a sufficient amount of test code. One easy way to handle this is to write test code when you develop a part of a feature.

zkchang97 committed:
Fixed AddRoom functions
Merge remote-tracking branch 'upstream/master'

# Conflicts:
#	src/main/java/booking/BookingList.java
#	src/main/java/command/Command.java
#	src/main/java/control/Parser.java
#	src/main/java/room/RoomList.java

Alex-Teo committed:
Merge pull request #70 from zkchang97/master

[CS2113T-W12-3]-Zhong Kai-Fixed AddRoom functions

Alex-Teo committed:
Merge pull request #10 from AY1920S1-CS2113T-W12-3/master

merge 11

zkchang97 committed:
Adds JavaDoc for Room commands

zkchang97 committed:
Merge pull request #71 from zkchang97/master

Adds JavaDoc for Room commands

AmosChan97 committed:
List Bookings

AmosChan97 committed:
Merge remote-tracking branch 'remotes/upstream/master'

Alex-Teo committed:
Merge pull request #11 from AY1920S1-CS2113T-W12-3/master

Merge 12

zkchang97 committed:
Adds JavaDoc for AddRoom.java

zkchang97 committed:
Merge pull request #72 from zkchang97/master

Adds JavaDoc for AddRoom.java

Alex-Teo committed:
Merge pull request #12 from AY1920S1-CS2113T-W12-3/master

merge 13

Alex-Teo committed:
Add Javadoc comments to booking related classes

Alex-Teo committed:
Merge pull request #73 from Alex-Teo/master

[CS2113T-W12-3]-Alex-Teo-Add-JavaDoc-Comments-for-Booking-type-classes

AmosChan97 committed:
List Bookings

AmosChan97 committed:
Merge remote-tracking branch 'remotes/upstream/master'

AmosChan97 committed:
Delete Bookings

AmosChan97 committed:
Delete Bookings

AmosChan97 committed:
Merge pull request #74 from AmosChan97/master

Delete Command and other cleanup

Alex-Teo committed:
Merge pull request #13 from AY1920S1-CS2113T-W12-3/master

merge 13

Alex-Teo committed:
Add authors to classes

Alex-Teo committed:
Merge pull request #75 from Alex-Teo/master

Add-Authors

AmosChan97 committed:
Delete Bookings

AmosChan97 committed:
Merge remote-tracking branch 'remotes/upstream/master'

wyinnnn committed:
Merge remote-tracking branch 'remotes/upstream/master'

# Conflicts:
#	src/main/java/Parser.java

AmosChan97 committed:
ListCommand

AmosChan97 committed:
ListCommand

AmosChan97 committed:
Merge pull request #77 from AmosChan97/master

ListCommand + other fixes

Alex-Teo committed:
Merge pull request #14 from AY1920S1-CS2113T-W12-3/master

Merge 15

Alex-Teo committed:
Add listbooking for monthly and daily commands, and travis support

Alex-Teo committed:
travis support with correct email

Alex-Teo committed:
Updated according to checkstyle

Alex-Teo committed:
Removed travis for a bit

Alex-Teo committed:
Author add

Alex-Teo committed:
Merge pull request #78 from Alex-Teo/master

[CS2113T-W12-3]-Alex Teo-CheckStyle commits and other commands

Alex-Teo committed:
Add test cases

zkchang97 committed:
Adds JavaDoc for AddRoom.java

zkchang97 committed:
Adds JavaDoc for DeleteRoomCommand.java

AmosChan97 committed:
fixed storage for bookings

AmosChan97 committed:
Merge pull request #79 from zkchang97/master

Adds DeleteRoomCommand.java & JavaDocs

AmosChan97 committed:
Merge remote-tracking branch 'remotes/upstream/master'

Alex-Teo committed:
Merge pull request #80 from AmosChan97/master

Fixed dateTime storage. Changes to list command

Alex-Teo committed:
Add exit ability

Alex-Teo committed:
Add exit ability

wyinnnn committed:
Merge remote-tracking branch 'remotes/upstream/master'

# Conflicts:
#	src/main/java/control/Parser.java
#	src/main/java/storage/Constants.java

wyinnnn committed:
fixed some errors

Alex-Teo committed:
Merge branch 'AY1920S1-CS2113T-W12-3-master'

Alex-Teo committed:
Made changes to DateTime

wyinnnn committed:
added Inventory

Alex-Teo committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-W12-3/main into AY1920S1-CS2113T-W12-3-master

                                      # Conflicts:
                                      #	.travis.yml
                                      #	data/roomlist.txt
                                      #	src/main/java/booking/Booking.java
                                      #	src/test/java/AddBookingTest1.java
                                      Conflicts settled

Alex-Teo committed:
Remerge

Alex-Teo committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-W12-3/main into AY1920S1-CS2113T-W12-3-master

Alex-Teo committed:
Merge branch 'AY1920S1-CS2113T-W12-3-master'

zkchang97 committed:
Changed formatting to add rooms

Alex-Teo committed:
Merge pull request #81 from wyinnnn/master

Inventory

Alex-Teo committed:
Merge branch 'master' into master

Alex-Teo committed:
Merge pull request #82 from zkchang97/master

Changed formatting to add rooms

Alex-Teo committed:
Merge pull request #16 from AY1920S1-CS2113T-W12-3/master

merge 16b

zkchang97 committed:
Adds constants

zkchang97 committed:
Merge pull request #83 from zkchang97/master

Adds constants for Room

Alex-Teo committed:
checkstyle correction

Alex-Teo committed:
Merge branch 'master' into master

Alex-Teo committed:
Merge pull request #17 from AY1920S1-CS2113T-W12-3/master

Merge 17

Alex-Teo committed:
checkstyle and Test correction

Alex-Teo committed:
Merge pull request #84 from Alex-Teo/master

Checkstyle fix

zkchang97 committed:
Adds author labels

zkchang97 committed:
Merge remote-tracking branch 'upstream/master'

AmosChan97 committed:
minor changes to fxml and commands

AmosChan97 committed:
Merge remote-tracking branch 'remotes/upstream/master'

# Conflicts:
#	src/main/java/command/DeleteRoomCommand.java
#	src/main/java/ui/ListBox.java

Commit:
Merge pull request #8 from AY1920S1-CS2113T-W12-3/master

update personal repo

Commit:
Reset

Commit:
Merge remote-tracking branch 'origin/master'

AmosChan97 committed:
GUI for ListRoom command

Commit:
Create AddUserCommand, RemoveUserCommand, LoginCommand, LogoutCommand
Update User Package

Commit:
Change format for storage initializing

Commit:
Merge pull request #86 from AmosChan97/master

GUI for ListRoom command, and fixed storage for rooms

Commit:
Merge pull request #87 from AmirAzhar/master

Create AddUserCommand, RemoveUserCommand, LoginCommand, LogoutCommand

Commit:
Merge pull request #10 from AY1920S1-CS2113T-W12-3/master

Update Personal Repo

zkchang97 committed:
Merge remote-tracking branch 'upstream/master'

AmosChan97 committed:
edit, reject and approve

AmosChan97 committed:
Merge remote-tracking branch 'remotes/upstream/master'

# Conflicts:
#	src/main/java/command/ApproveCommand.java

Alex-Teo committed:
Merge pull request #18 from AY1920S1-CS2113T-W12-3/master

Merge 17

zkchang97 committed:
Merge pull request #88 from zkchang97/master

Adds author labels

AmosChan97 committed:
travis

Alex-Teo committed:
Fix ListMonth and ListYear Commands,

Alex-Teo committed:
Merge pull request #89 from AmosChan97/master

travis setup

AmosChan97 committed:
updated Storage

Alex-Teo committed:
Merge branch 'master' into master

Alex-Teo committed:
Merge pull request #19 from AY1920S1-CS2113T-W12-3/master

Merge 18

Alex-Teo committed:
Updated

AmosChan97 committed:
Merge remote-tracking branch 'remotes/upstream/master'

AmosChan97 committed:
updated Storage

AmosChan97 committed:
Merge pull request #90 from AmosChan97/master

Updated storage

Alex-Teo committed:
Merge pull request #20 from AY1920S1-CS2113T-W12-3/master

merge 19

Commit:
Merge pull request #11 from AY1920S1-CS2113T-W12-3/master

Update personal repo

Commit:
Fix bugs with user commands:
- unable logout
- ability to login when already logged in
- username accepts spaces
- remove case-sensitive issue for username

zkchang97 committed:
Adds help list

AmosChan97 committed:
Current User Label and Approved By Box

AmosChan97 committed:
Merge pull request #127 from AmirAzhar/master

Fix user bugs

AmosChan97 committed:
Merge remote-tracking branch 'remotes/upstream/master'

# Conflicts:
#	src/main/java/command/LoginCommand.java
#	src/main/java/command/LogoutCommand.java

zkchang97 committed:
Merge remote-tracking branch 'upstream/master'

Commit:
Merge pull request #128 from AmosChan97/master

Current User Label and Approved By Box

Commit:
Merge pull request #12 from AY1920S1-CS2113T-W12-3/master

Update personal repo

Alex-Teo committed:
Merge pull request #21 from AY1920S1-CS2113T-W12-3/master

Merge 19

zkchang97 committed:
Adds help list

zkchang97 committed:
Merge remote-tracking branch 'upstream/master'

# Conflicts:
#	src/main/java/ui/Ui.java

AmosChan97 committed:
Approve GUI

zkchang97 committed:
Adds help command list

AmosChan97 committed:
Allowance for spaces

AmosChan97 committed:
Merge pull request #129 from zkchang97/master

Adds help command

AmosChan97 committed:
Merge remote-tracking branch 'remotes/upstream/master'

# Conflicts:
#	src/main/java/control/Parser.java

AmosChan97 committed:
Merge pull request #130 from AmosChan97/master

Allowance for spaces + font size

zkchang97 committed:
Edit Checkstyle

zkchang97 committed:
Merge remote-tracking branch 'upstream/master'

# Conflicts:
#	src/main/java/command/AddBookingCommand.java

zkchang97 committed:
Merge pull request #132 from zkchang97/master

Edit Checkstyle

Alex-Teo committed:
Merge pull request #22 from AY1920S1-CS2113T-W12-3/master

merge 20

Alex-Teo committed:
Fix checkstyle

Alex-Teo committed:
Merge pull request #133 from Alex-Teo/master

[CS2113T-W12-3]-Alex Teo-Fix Checkstyle

Commit:
Merge pull request #13 from AY1920S1-CS2113T-W12-3/master

update personal repo

Alex-Teo committed:
fixed changes

Alex-Teo committed:
Merge pull request #134 from Alex-Teo/master

fixed changes

Commit:
Merge pull request #14 from AY1920S1-CS2113T-W12-3/master

update personal repo

AmosChan97 committed:
Allow some live updates
Prevent not logged in from approving or rejecting bookings

AmosChan97 committed:
Merge remote-tracking branch 'remotes/upstream/master'

# Conflicts:
#	src/main/java/command/AddBookingCommand.java
#	src/main/java/command/ListBookingMonthCommand.java
#	src/main/java/control/Parser.java
#	src/main/java/ui/Ui.java
#	src/test/java/AddBookingTest1.java

AmosChan97 committed:
Add borders for RoomListBox

AmosChan97 committed:
Removed commented lines

AmosChan97 committed:
Merge pull request #135 from AmosChan97/master

Live Updates for list

Alex-Teo committed:
Merge pull request #23 from AY1920S1-CS2113T-W12-3/master

merge 21

Alex-Teo committed:
Added storagemanager

zkchang97 committed:
Merge remote-tracking branch 'upstream/master'

# Conflicts:
#	src/main/java/command/AddBookingCommand.java

zkchang97 committed:
Merge remote-tracking branch 'upstream/master'

AmosChan97 committed:
Travis

AmosChan97 committed:
Merge pull request #136 from Alex-Teo/master

Add storagemanager

AmosChan97 committed:
Merge remote-tracking branch 'remotes/upstream/master'

zkchang97 committed:
Merge remote-tracking branch 'upstream/master'

Alex-Teo committed:
Fixed ListDay and checkBooking

AmosChan97 committed:
Merge pull request #137 from Alex-Teo/master

Fixed ListDay and checkBooking

AmosChan97 committed:
default rooms

AmosChan97 committed:
Merge remote-tracking branch 'remotes/upstream/master'

# Conflicts:
#	src/main/java/booking/BookingList.java

AmosChan97 committed:
checkBooking

AmosChan97 committed:
Merge pull request #138 from AmosChan97/master

Fixed Check Booking

Alex-Teo committed:
Merge pull request #24 from AY1920S1-CS2113T-W12-3/master

merge 23

zkchang97 committed:
Merge remote-tracking branch 'upstream/master'

Commit:
Merge pull request #15 from AY1920S1-CS2113T-W12-3/master

update personal repo

Commit:
Merge pull request #139 from AmirAzhar/master

Fix bug in RemoveUserCommand

Commit:
Fix bugs with RemoveUserCommand:
- ability to remove usernames that are made up of more than one word, separated by spaces
- minor code adjustments

Commit:
Merge pull request #140 from AmirAzhar/master

Fix bugs with RemoveUserCommand

Commit:
Minor code fixes and add test cases

Commit:
Merge pull request #141 from AmirAzhar/master

Minor code fixes and add test cases

Commit:
Minor code fixes

Commit:
Merge pull request #142 from AmirAzhar/master

Minor code fixes

zkchang97 committed:
Merge remote-tracking branch 'upstream/master'

wyinnnn committed:
added ListInventory command
added InventorySorter
updated Inventory, Item with rooms

zkchang97 committed:
Adds AddRoom Testing

Commit:
Merge pull request #143 from zkchang97/master

Adds AddRoom Testing

Commit:
Minor code fixes

Commit:
Merge branch 'master' into master

Commit:
Merge pull request #144 from AmirAzhar/master

Minor code fixes

zkchang97 committed:
Adds addroom clashes

zkchang97 committed:
Merge remote-tracking branch 'upstream/master'

zkchang97 committed:
Merge pull request #145 from zkchang97/master

Adds addroom clashes

Commit:
Add author

Commit:
Merge remote-tracking branch 'origin/master'

# Conflicts:
#	src/main/java/command/AddUserCommand.java
#	src/main/java/command/LoginCommand.java
#	src/main/java/command/LogoutCommand.java
#	src/main/java/command/RemoveUserCommand.java
#	src/main/java/command/UnusedCreateAccountCommand.java
#	src/main/java/command/UnusedLoginCommand.java
#	src/main/java/command/UnusedLogoutCommand.java

Commit:
Add author

Commit:
Merge pull request #146 from AmirAzhar/master

Add author

Alex-Teo committed:
Merge pull request #25 from AY1920S1-CS2113T-W12-3/master

merge 24

AmosChan97 committed:
minor changes

AmosChan97 committed:
Merge remote-tracking branch 'remotes/upstream/master'

zkchang97 committed:
More tests

zkchang97 committed:
Merge branch 'master' into master

zkchang97 committed:
Merge pull request #147 from zkchang97/master

More tests

Alex-Teo committed:
Merge pull request #26 from AY1920S1-CS2113T-W12-3/master

Merge 26

Alex-Teo committed:
Add Exception test for add, approve, reject and edit (do not have for out of bounds)

Alex-Teo committed:
Merge pull request #148 from Alex-Teo/master

Added exception tests for booking related commands

Commit:
minor bug fixes

Commit:
Merge pull request #150 from AmirAzhar/master

minor bug fixes

wyinnnn committed:
updated inventory function

AmosChan97 committed:
changes to UI

AmosChan97 committed:
Merge remote-tracking branch 'remotes/upstream/master'

wyinnnn committed:
Merge remote-tracking branch 'remotes/upstream/master'

# Conflicts:
#	src/main/java/command/ListInventoryCommand.java
#	src/main/java/storage/Constants.java

Commit:
Merge pull request #16 from AY1920S1-CS2113T-W12-3/master

update personal repo

zkchang97 committed:
Merge pull request #151 from wyinnnn/master

Added and updated Inventory commands

zkchang97 committed:
Changes made to Help message

zkchang97 committed:
Merge pull request #152 from zkchang97/master

Changes made to Help message

Commit:
Merge pull request #17 from AY1920S1-CS2113T-W12-3/master

update personal repo

Alex-Teo committed:
Merge pull request #27 from AY1920S1-CS2113T-W12-3/master

merge 27

AmosChan97 committed:
minor change

AmosChan97 committed:
Merge remote-tracking branch 'remotes/upstream/master'

zkchang97 committed:
Fixed bug when saving to inventory.txt

zkchang97 committed:
Merge pull request #153 from zkchang97/master

Fixed bug when saving to inventory.txt

Alex-Teo committed:
Merge pull request #28 from AY1920S1-CS2113T-W12-3/master

merge 28

Alex-Teo committed:
Fixed stuff, added real time for find and findindex

Alex-Teo committed:
Merge pull request #154 from Alex-Teo/master

Add real time for Find and FindIndex

zkchang97 committed:
Changes made to error message

zkchang97 committed:
Merge pull request #155 from zkchang97/master

Changes made to error message

Alex-Teo committed:
Merge pull request #29 from AY1920S1-CS2113T-W12-3/master

merge 29

Alex-Teo committed:
Checkstyle fixes, adn added list error cases

Alex-Teo committed:
Merge pull request #156 from Alex-Teo/master

Add list error test cases

zkchang97 committed:
Changes made to error message

zkchang97 committed:
Merge branch 'master' into master

AmosChan97 committed:
add JUnit test for Ui

AmosChan97 committed:
Merge remote-tracking branch 'remotes/upstream/master'

# Conflicts:
#	src/test/java/AddBookingTest4.java

Commit:
Merge pull request #18 from AY1920S1-CS2113T-W12-3/master

update personal repo

AmosChan97 committed:
Modified rooms
Added capacity

Commit:
minor code fixes
- added .trim() where appropriate

wyinnnn committed:
inventorybox

Commit:
Merge pull request #159 from AmirAzhar/master

minor code fixes

AmosChan97 committed:
Allow spaces
Updated JUnit

AmosChan97 committed:
Allow spaces
checkstyle

AmosChan97 committed:
Merge pull request #158 from AmosChan97/room

Room

AmosChan97 committed:
Merge pull request #157 from zkchang97/master

Changes made to error message

zkchang97 committed:
Merge remote-tracking branch 'upstream/master'

# Conflicts:
#	src/main/java/command/AddInventoryCommand.java

zkchang97 committed:
Merge remote-tracking branch 'upstream/master'

AmosChan97 committed:
Parser test

Alex-Teo committed:
Merge pull request #30 from AY1920S1-CS2113T-W12-3/master

Merge 30

AmosChan97 committed:
checkstyle

AmosChan97 committed:
Merge pull request #160 from AmosChan97/testcode

Testcode

Alex-Teo committed:
Error message fixes

zkchang97 committed:
Merge remote-tracking branch 'upstream/master'

# Conflicts:
#	src/main/java/command/AddInventoryCommand.java

zkchang97 committed:
Merge remote-tracking branch 'upstream/master'

Alex-Teo committed:
Add exception error for list month, updated help command

Alex-Teo committed:
Merge pull request #161 from Alex-Teo/master

Updated Help, Add more exceptions for ListMonthCommand

zkchang97 committed:
Merge remote-tracking branch 'upstream/master'

Alex-Teo committed:
Update ListTest.java

zkchang97 committed:
Small changes to error messages

zkchang97 committed:
Merge pull request #162 from zkchang97/master

Small changes to error messages

wyinnnn committed:
inventory tests

wyinnnn committed:
Merge remote-tracking branch 'upstream/master'

AmosChan97 committed:
Merge pull request #163 from wyinnnn/master

inventory tests

AmosChan97 committed:
Update Docs

AmosChan97 committed:
Merge pull request #164 from AmosChan97/master

Update Docs

AmosChan97 committed:
Update README.md

Alex-Teo committed:
Merge pull request #31 from AY1920S1-CS2113T-W12-3/master

Merge 31

Alex-Teo committed:
Add exception error for edit

Alex-Teo committed:
Merge pull request #165 from Alex-Teo/master

Add edit exception error
